components/ButtonCustom.js

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import { StyleSheet, Text, View, TouchableHighlight } from "react-native"
+
+export const ButtonCustom = ({ action, name }) => {
+  return (
+    <TouchableHighlight style={styles.touch} onPress={() => action()}>
```

Comment:
I think it is cleaner like this, but no changes are required here.
```suggestion
    <TouchableHighlight style={styles.touch} onPress={action}>
```